muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/activity-badge.scss

```diff
@@ -0,0 +1,50 @@
+@import "../base/colors";
+@import "../base/mixins";
+@import "../base/vars";
+@import "../base/breakpoints";
+
+$progress-units: 100;
+
+.activity-badge {
+  display: flex;
+  height: 30px;
+  justify-content: space-around;
+  width: 30px;
+}
+
+.activity-badge__item {
+  align-items: flex-end;
+  background-color: $color-default;
+  display: flex;
+  height: 100%;
+  width: 10px;
+}
+
+.activity-badge__item--assignment {
+  border: 1px solid $color-assignment;
+  
+  .activity-badge__unit-bar {
+    background-color: $color-assignment;   
+  }
+}
+
+.activity-badge__item--exercise {
+  border: 1px solid $color-exercise;
+    
+  .activity-badge__unit-bar {
+    background-color: $color-exercise;   
+  }
+}
+
+.activity-badge__unit-bar {
+  background-color: $color-default-contrast;
+  width:100%;
+}
+
+@for $i from 1 through $progress-units {
+    $height: $i + %;
```

Comment:
It is the only way it will work, dear karvatura

Reply:
We should use percentage function provided by sass instead of defining number as a string when interpolating as we can't add the unit directly after $i variable. Hound is complaining about failed calculation basically. If you add **1px** then its a calculation and if you add **bold** its transformed as a string like mentioned in the e.g part. 

If you do it like this `$height: $i + %` the $height will be a string that looks like a number instead of a number.

This should be the correct way according the interwebs:
```
$progress-units: 100;

@for $i from 1 through $progress-units {
  
  .activity-badge__unit-bar--#{$i} {
    height: percentage($i/100);
  }
}
```